content/blog/2022/10/env-var-secrets.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Scanning secrets in environment variables"
+authors:
+- Raul Cabello Martin
+date: 2022-10-17
+---
+
+We are thrilled to announce you can now scan your environment variables for secrets with the new
+[env-variable-secrets-scanner-policy](https://github.com/kubewarden/env-variable-secrets-scanner-policy)! This policy rejects a `Pod`
+or [workload resources](https://kubernetes.io/docs/concepts/workloads/) such as `Deployments`, `ReplicaSets`, `DaemonSets`
+, `ReplicationControllers`, `Jobs`, `CronJobs` etc. if a secret is found in the environment variable
+within a container, init container, or ephemeral container.
```

Comment:
Please add a paragraph explaining how can find secrets that are leaked inside of base64 encoded env variables

Reply:
added in https://github.com/kubewarden/kubewarden.io/pull/138/commits/f2ff7c000fd9c19a0ee8e5b5e60079ed760fcbba

Reply:
thanks!